devel/swig/Portfile

```diff
@@ -47,6 +47,14 @@ test.target     check
 
 configure.args-append --without-javascript
 
+subport swig-java {
```

Comment:
Please more the `swig-java` subports to where subports are listed in the Portfiles, respectively. Otherwise LGTM & thanks!

Reply:
Done in db282671aef

Reply:
perfect; thx!